hipscat/util.py

```diff
@@ -229,22 +233,17 @@ def which_cull_and_pixorder(c1, c2):
         flag that catalog to be culled to the smaller order
     '''
 
-    c1_order = int(c1.split('Norder')[1].split('/')[0])
-    c1_pix = int(c1.split('Npix')[1].split('/')[0])
-    c2_order = int(c2.split('Norder')[1].split('/')[0])
-    c2_pix = int(c2.split('Npix')[1].split('/')[0])    
-
-    tocull1 = False
-    tocull2 = False
```

Comment:
If this doesn't do the culling any more, should the method be renamed?

Reply:
Method has been renamed